bats/foreman_helper.bash

```diff
@@ -17,3 +17,13 @@ tForemanVersion() {
     fi
   ) | cut -d. -f1-2
 }
+
+Pulp2() {
+  tPackageExists pulp-server
+}
+
+tSkipIfNoPulp2() {
+  if ! tIsPulp2; then
+   skip "Puppet content not available with Pulp 3 only"
```

Comment:
You're using this in more cases than Puppet, so we either should have a more generic message here, or multiple `skip` functions with different explicit messages.

Reply:
Now it supports multiple cases.